src/main/java/com/epam/Main.java

```diff
@@ -0,0 +1,7 @@
+package com.epam;
```

Comment:
We don't need this class

Reply:
Probably I understand it the wrong way, thought that something should be here as a stub. Ok, I will delete it.